src/main/java/dumpling/Ui.java

```diff
@@ -0,0 +1,45 @@
+package dumpling;
+
+import java.util.Scanner;
+
+public class Ui {
+
+    private static final String DIVIDER = "    ____________________________________________________________";
+
+    private Scanner scanner;
+
+    public Ui() {
+        this.scanner = new Scanner(System.in);
+    }
+
+    public String readCommand() {
+        return this.scanner.nextLine();
+    }
+
+    public void showError(DumplingException exception) {
+        this.echo("     " + exception.getMessage());
+    }
+
+    public void showLine() {
+        this.echo(DIVIDER);
+    }
+
+    public void showWelcome() {
```

Comment:
Missing JavaDoc comments for some public methods including this one! Minimally just a header comment shoud suffice.